src/main/java/com/schematica/client/renderer/chunk/overlay/RenderOverlay.java

```diff
@@ -178,32 +177,33 @@ private static void increamentRenderChunkedUpdated()
     private static int getSides(final Block block, final World world, final BlockPos pos, int s)
     {
         int sides = s;
-        if (block.shouldSideBeRendered(world, pos.offset(EnumFacing.DOWN), EnumFacing.DOWN))
+        IBlockState blockState = world.getBlockState(pos);
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Local variable 'blockState' could be declared final [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=pmd%3ALocalVariableCouldBeFinal)